packages/extension-api-explorer/README.md

```diff
@@ -22,11 +22,12 @@ The API Explorer extension can be manually installed and run with a Looker insta
          use_form_submit: yes
          use_embeds: yes
          core_api_methods: ["versions", "api_spec"]
-         external_api_urls: ["https://raw.githubusercontent.com"]
+         external_api_urls: ["https://raw.githubusercontent.com", "http://localhost:30000"]
          oauth2_urls: []
       }
    }
    ```
+   the `http://localhost:3000` is for when you want to use the [API Explorer file server](/apix-files/README.md)
```

Comment:
Be interesting to see how this works in marketplace. Do we want to put a dummy entry in so that user can configure hosts to point at it? Or just go with localhost:3000. 3000 pretty popular. Consider using non popular port

Reply:
whoops, documentation error. That's supposed to be `30000`